cypress/integration/common/selectors-and-assertions.spec.ts

```diff
@@ -52,7 +52,7 @@ export const whenIClickOnTheErrorSummaryLinkContaining = (
 };
 
 export const whenIType = (value: string, selector: string): void => {
-  cy.get(selector).clear().type(value);
+  cy.get(selector).should("be.empty").type(value);
 };
```

Comment:
Makes explicit that the user expects to be entering information in a pristine field.  Will error if the user is displayed existing data.